src/themes/hugo-universal-theme/static/js/front.js

```diff
@@ -427,4 +428,20 @@ function quoteAnimation() {
   const hiddenQuotes = document.querySelectorAll('.hide-quote');
   console.log(hiddenQuotes);
   hiddenQuotes.forEach((quote) => observer.observe(quote));
-}
\ No newline at end of file
+}
+function openAutoDetails() {
+  function openTarget() {
+    var hash = window.location.hash.substring(1);
+    if (hash) {
+      var summary = document.getElementById(hash);
+      if (summary && summary.tagName.toLowerCase() === 'summary') {
+        var details = summary.parentElement;
```

Comment:
consider [`summary.closest('details')`](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) to avoid having to do the parent stuff and tag check yourself